app/models/manageiq/providers/vmware/infra_manager/vm.rb

```diff
@@ -9,6 +9,8 @@ class ManageIQ::Providers::Vmware::InfraManager::Vm < ManageIQ::Providers::Infra
     unsupported_reason_add(:clone, _('Clone operation is not supported')) if blank? || orphaned? || archived?
   end
 
+  supports :reconfigure_disks
```

Comment:
I haven't find any. Before adding support for reconfigure_disks by rhevm, only vmware supported that feature and the dialog was the responsible to block any other provider from doing so. I didn't find any previous validation of it in code.